lnst/Devices/Device.py

```diff
@@ -1087,9 +1088,12 @@ def _get_vf_rep_devices(self):
 
     def _get_vf_count(self):
         # TODO: create sysfs api for Device, then run self.[get|set]_sysfs("device/sriov_numvfs")
-        stdout, _ = exec_cmd(f"cat /sys/class/net/{self.name}/device/sriov_numvfs")
-
-        return int(stdout)
+        try:
+            stdout, _ = exec_cmd(f"cat /sys/class/net/{self.name}/device/sriov_numvfs",
+                                 log_outputs = False)
+            return int(stdout)
+        except ExecCmdFail:
+            return 0
```

Comment:
alternatively i'd say we can also `raise DeviceFeatureNotSupported` so that the user of this may decide on if reading this is a MUST or if it is a NICE TO HAVE.

this may be useful when there's information difference between "vf_count == 0" and "vfs are NOT SUPPORTED".